addons/zeus/CfgEventHandlers.hpp

```diff
@@ -4,3 +4,11 @@ class Extended_PreInit_EventHandlers {
         init = QUOTE(call COMPILE_FILE(XEH_preInit));
     };
 };
+
+class Extended_InitPost_EventHandlers {
+    class AllVehicles {
+        class ADDON {
+            init = QUOTE(call FUNC(addObjectToCurator));
```

Comment:
Using only a call will use the `_this` from the scope where you use the call. `QFUNC` is not an option since we need to call the function here.